components/ModFacilitySubmissionLeftNavbar.vue

```diff
@@ -0,0 +1,161 @@
+            </button>
+            <button
+                data-testid="submission-form-leftNav-addresses"
+                :class="{ 'bg-lightBlue': activeSection === 'addresses', 'bg-transparent': activeSection !== 'addresses' }"
+                class="w-full py-4 my-2 text-sm text-start pl-2 rounded border-b-2 border-slate-200"
+                @click="scrollToSectionOfFacilityForm('addresses')"
+            >
+                {{ $t("modPanelFacilitySubmissionLeftNavbar.addresses") }}
+            </button>
+            <button
+                data-testid="submission-form-leftNav-googleMapsInformation"
+                :class="{ 'bg-lightBlue': activeSection === 'google-maps-information',
+                          'bg-transparent': activeSection !== 'google-maps-information' }"
+                class="w-full py-4 my-2 text-sm text-start pl-2 rounded border-b-2 border-slate-200"
+                @click="scrollToSectionOfFacilityForm('google-maps-information')"
+            >
+                {{ $t("modPanelFacilitySubmissionLeftNavbar.googleMapsInformation") }}
+            </button>
+            <button
+                data-testid="submission-form-leftNav-healthcareProfessionalIds"
+                :class="{ 'bg-lightBlue': activeSection === 'healthcare-professional-ids',
+                          'bg-transparent': activeSection !== 'healthcare-professional-ids' }"
+                class="w-full py-4 my-2 text-sm text-start pl-2 rounded border-b-2 border-slate-200"
+                @click="scrollToSectionOfFacilityForm('healthcare-professional-ids')"
+            >
+                {{ $t("modPanelFacilitySubmissionLeftNavbar.healthcareProfessionalIds") }}
+            </button>
+            <button
+                data-testid="submission-form-leftNav-changeLog"
+                :class="{ 'bg-lightBlue': activeSection === 'change-log', 'bg-transparent': activeSection !== 'change-log' }"
+                class="w-full py-4 my-2 text-sm text-start pl-2 rounded border-b-2 border-slate-200"
+                @click="scrollToSectionOfFacilityForm('change-log')"
+            >
+                {{ $t("modPanelFacilitySubmissionLeftNavbar.changeLog") }}
+            </button>
+        </div>
+    </div>
+</template>
+
+<script setup lang="ts">
+import { ref, type Ref, onMounted, onUnmounted } from 'vue'
+import { useModerationSubmissionsStore } from '~/stores/moderationSubmissionsStore'
+
+const moderationSubmissionsStore = useModerationSubmissionsStore()
+const activeSection: Ref<string> = ref('contact-information')
+const isScrolling: Ref<boolean> = ref(false)
+
+const scrollToSectionOfFacilityForm = (sectionId: string) => {
+    document.getElementById(sectionId)?.scrollIntoView({
+        behavior: 'smooth'
+    })
+    activeSection.value = sectionId
+
+    isScrolling.value = true
+
+    setTimeout(() => {
+        isScrolling.value = false
+    }, 1000)
+}
+
+const observeFormSections = () => {
```

Comment:
this is pretty cool! really adds to the user experience ✨ 
See the comment below for an improvement idea